locksmith/src/controllers/eventController.ts

```diff
@@ -0,0 +1,43 @@
+// eslint-disable-next-line no-unused-vars, import/no-unresolved
+import { Request, Response } from 'express-serve-static-core'
+import Normalizer from '../utils/normalizer'
+
+const Sequelize = require('sequelize')
+
+const models = require('../models')
+
+const { Event } = models
+const Op = Sequelize.Op
+
+namespace EventController {
+  export const create = async (req: Request, res: Response): Promise<any> => {
+    let event = req.body.message.event
+
+    try {
+      event.owner = Normalizer.ethereumAddress(event.owner)
+      event.lockAddress = Normalizer.ethereumAddress(event.lockAddress)
+      await Event.create(event)
+
+      return res.sendStatus(200)
+    } catch (e) {
+      return res.sendStatus(409)
+    }
+  }
+
+  export const find = async (req: Request, res: Response): Promise<any> => {
+    let lockAddress = req.params.lockAddress
+    let event = await Event.findOne({
```

Comment:
Would you be ok to add an event Operations? I think this would be more consistent with the rest of locksmith?

Reply:
yeah, we can give it a go. I was a bit reluctant to do it this early as both of these methods are calling Sequelize methods without a ton of value add at this point